prdoc/pr_2061.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add Parameters Pallet
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Adds `pallet-parameters` that allows to have parameters for pallet configs that dynamically change at runtime. Allows to be permissioned on a per-key basis and is compatible with ORML macros.
+
+crates:
```

Comment:
Is this list of crates provided here used in a systematic way already? or is it just some soft hint?

Reply:
Its not used yet. We dont even validate for typos.